lab_3_ann_retriever/main.py

```diff
@@ -75,6 +98,13 @@ def load_vector(state: dict) -> Vector | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not (isinstance(state, dict) and "len" in state and "elements" in state and isinstance(
```

Comment:
`if "len" in state` -> `return None` ?

Reply:
Strange check at all

Reply:
there is "if NOT ( ... and "len" in state and ...)"